test/parallel/test-stream-readable-async-iterators.js

```diff
@@ -396,6 +396,26 @@ async function tests() {
     }
   }
 
+  console.log('readable side of a transform stream pushes null');
+  {
```

Comment:
Nit: ideally, the `console.log` should be moved inside of the code block to be consistent with the other test cases.